### PR TITLE
UX-1101/Storage details

### PR DESCRIPTION
**Pull request by jruddell**
### Overview Page
image

### Yaml Page
I also addressed the styling so this should look the same everywhere this generic entity page is used
image

Todo:
- [ ] need to add events page
- [ ] need to fix transient states so there is feedback to the user.
- [ ] need to convert to yaml instead of rendering JSON
- [ ] need to validate the yaml

### CODE REVIEW

**Review by rajshekhar23 — APPROVED** [review]
For me everything looks good !

File: src/api-client/Qbert.ts [hunk 1]
```diff
@@ -854,6 +872,25 @@ class Qbert extends ApiService {
     return data.items.map(this.convertResource(clusterId))
   }
 
+  @trackApiMethodMetadata({
+    url: '/clusters/:clusterId/k8sapi/apis/storage.k8s.io/v1/storageclasses/:name',
+    type: 'PUT',
+    params: ['clusterId'],
```
Comment (knnguy) [inline src/api-client/Qbert.ts:878]: Missing storage class name as a param here

File: src/api-client/Qbert.ts [hunk 2]
```diff
@@ -854,6 +872,25 @@ class Qbert extends ApiService {
...
+    error: { isClusterError: true, k8sResource: 'StorageClass' },
+  })
+  updateClusterStorageClasses = async (clusterId, name, body) => {
```
Comment (knnguy) [inline src/api-client/Qbert.ts:881]: Shouldn't this be singular? updateClusterStorageClass

File: src/app/plugins/kubernetes/components/services/overview.tsx [hunk 3]
```diff
@@ -164,7 +166,12 @@ const ServiceOverview = ({ service }) => {
   return (
     <EntityOverviewPage headerCards={headerCards}>
       {/* Selectors Table */}
-      <Text className={classes.title} variant="subtitle2" component="h3">
+      <LabelsAndAnnotationsTable
+        title="Selectors"
+        emptyMessage="No selectors"
+        labelRows={selectorRows}
+      />
+      {/* <Text className={classes.title} variant="subtitle2" component="h3">
```
Comment (knnguy) [inline src/app/plugins/kubernetes/components/services/overview.tsx:174]: We don't need this anymore, right?

**Review by knnguy — APPROVED** [review]

File: src/api-client/Qbert.ts (hunk 2 above)
Comment (jruddell) [reply src/api-client/Qbert.ts:881]: Isn't it already singular?

**Pull request merged**